test/controllers/calls_controller_test.rb

```diff
@@ -0,0 +1,51 @@
+require 'test_helper'
+
+class CallsControllerTest < ActionController::TestCase
+  include Devise::TestHelpers
+
+  def setup
+    @user = build :user
+    sign_in @user
+    @case = create :case
+    @call = build :call
+  end
+
+  describe 'create method' do
+    before do
+      @call = attributes_for :call
+    end
+
+    it 'should create and save a new call' do
+      assert_difference 'Case.find(@case).calls.count', 1 do
+        post :create, status: @call.status, id: @case
```

Comment:
After adding 'validates_presence_of :status' to the call model, this test wasn't working with call: @call, but I think I've found the problem! @call was being defined in the setup and the before action. I took it out of the setup and changed the post method params back to what we originally had.